python_program/automatic_walk_time_tables/find_walk_table_points.py

```diff
@@ -4,6 +4,13 @@
 from gpxpy.gpx import GPXTrackPoint
 from typing import List, Tuple
 
+# Set up logging
+from . import log_helper
+import logging
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+handler.setFormatter(log_helper.Formatter())
+logger.addHandler(handler)
```

Comment:
Is it really necessary to the STeramHandler and Formater in a very file? Aren't you using the same logger object anyway by calling `logging.getLogger(__name__)`?

Reply:
Done.